Objects/complexobject.c

```diff
@@ -684,46 +684,58 @@ complex_float(PyObject *v)
     return NULL;
 }
 
+/*[clinic input]
+complex.conjugate
+
+complex.conjugate() -> complex
```

Comment:
How the output of help() looks now? Is not the signature is duplicated?

Reply:
## Master
```
Help on built-in function conjugate:

conjugate(...) method of builtins.complex instance
    complex.conjugate() -> complex

    Return the complex conjugate of its argument. (3-4j).conjugate() == 3+4j.
```

## PR
```
Help on built-in function conjugate:

conjugate() method of builtins.complex instance
    complex.conjugate() -> complex

    Return the complex conjugate of its argument. (3-4j).conjugate() == 3+4j.
````